src/v/cloud_storage/partition_manifest.h

```diff
@@ -128,10 +128,6 @@ class partition_manifest final : public base_manifest {
     bool add(const key& key, const segment_meta& meta);
     bool add(const segment_name& name, const segment_meta& meta);
 
-    /// Remove segment from the manifest only if the metadata matches
-    bool remove(const key& key, const segment_meta& meta);
```

Comment:
I think that belongs to the first commit

Reply:
fixed